common/settings.h

```diff
@@ -96,6 +96,7 @@ struct Settings {
   bool verbose_logging = false;
   std::string log_tag = "flutter";
   std::string icu_data_path;
+  std::string icu_symbol;
```

Comment:
Can you make this a `MappingCallback` instead. That way, the embedders can setup the mapping from whatever source they choose.

Reply:
done